### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by atarkowska**
To make clear that Django upgrade is required.

### CODE REVIEW

File: omero/sysadmins/server-upgrade.txt [hunk 1]
```diff
@@ -59,6 +59,13 @@ your previous version of OMERO. We recommend using a future-proof symlink
 if possible, so that these stanzas do not need updating with each OMERO
 server upgrade.
 
+.. note:: Since OMERO 5.2, the OMERO web framework no longer bundles
```
Comment (hflynn) [inline omero/sysadmins/server-upgrade.txt]: OMERO.web for consistency, otherwise this formats and reads well.

Comment (atarkowska) [inline omero/sysadmins/server-upgrade.txt]: Do you want me to update across entire doc? This bit was copied over from another location.

**Comment by mtbc** [comment]
See note at http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/server-upgrade.html#omero-web-configuration.

File: omero/sysadmins/server-upgrade.txt [hunk 2]
```diff
@@ -59,6 +59,13 @@ your previous version of OMERO. We recommend using a future-proof symlink
 if possible, so that these stanzas do not need updating with each OMERO
 server upgrade.
 
+.. note:: Since OMERO 5.2, the OMERO.web framework no longer bundles
+    a copy of the Django package, instead manual installation of
+    the Django dependency is required. It is highly recommended to use
+    `Django 1.8`_ (LTS) which requires Python 2.7. For more information
+    see :ref:`python-requirements` on the
+    :doc:`/sysadmins/version-requirements` page.
```
Comment (mtbc) [inline omero/sysadmins/server-upgrade.txt:67]: I'm wondering if we need separate links both to a section and to the page the section's on? Though if you think it adds value I'm fine with it.

**Comment by mtbc** [comment]
I wonder if we (should) have a card noting where in docs we mention version numbers and other such stuff that might need routine updates as things change.

**Comment by mtbc** [comment]
This PR seems fine to me to merge, just one possible quibble noted.

**Comment by hflynn** [comment]
I have a card to replace version numbers with variables set in the conf.py already (https://trello.com/c/uripejjj/394-add-min-version-for-postgres-etc-to-conf-py) so that in future only that file will need updating. Hopefully get on to it for the next release or for 5.3 at the latest.

File: omero/sysadmins/server-upgrade.txt (hunk 2 above)
Comment (hflynn) [inline omero/sysadmins/server-upgrade.txt:67]: I can't say I think it hurts, readers are of course free to ignore hyperlinks. I think it's good to reference the page and by the time we've mentioned it, we may as well link too.

**Pull request merged**